ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/Tables/CVEsTable.tsx

```diff
@@ -0,0 +1,177 @@
+                    </Th>
+                    <Th tooltip="Highest CVSS score of this CVE across images">Top CVSS</Th>
+                    <Th tooltip="Ratio of total environment affect by this CVE">
+                        Affected images
+                        {isFiltered && <DynamicColumnIcon />}
+                    </Th>
+                    <Th tooltip="Time since this CVE first affected an entity">First discovered</Th>
+                </Tr>
+            </Thead>
+            {cves.map(
+                (
+                    {
+                        cve,
+                        // summary,
+                        affectedImageCountBySeverity,
+                        topCVSS,
+                        affectedImageCount,
+                        firstDiscoveredInSystem,
+                    },
+                    rowIndex
+                ) => {
+                    const isExpanded = expandedRowSet.has(cve);
+
+                    return (
+                        <Tbody
+                            key={cve}
+                            style={{
+                                borderBottom: '1px solid var(--pf-c-table--BorderColor)',
+                            }}
+                            isExpanded={isExpanded}
+                        >
+                            <Tr>
+                                <Td
+                                    expand={{
+                                        rowIndex,
+                                        isExpanded,
+                                        onToggle: () => expandedRowSet.toggle(cve),
+                                    }}
+                                />
+                                <Td>
+                                    <Button
+                                        variant={ButtonVariant.link}
+                                        isInline
+                                        component={LinkShim}
+                                        href={getEntityPagePath('CVE', cve)}
+                                    >
+                                        {cve}
+                                    </Button>
+                                </Td>
+                                <Td>
+                                    <SeverityCountLabels
+                                        critical={affectedImageCountBySeverity.critical}
+                                        important={affectedImageCountBySeverity.important}
+                                        moderate={affectedImageCountBySeverity.moderate}
+                                        low={affectedImageCountBySeverity.low}
+                                    />
+                                </Td>
+                                {/* TODO: score version? */}
+                                <Td>{topCVSS.toFixed(1)}</Td>
+                                <Td>
+                                    {/* TODO: fix upon PM feedback */}
```

Comment:
👀  There are similar sections in the summary cards on other pages, please LMK what you get for feedback on this.